core/openapi/parameters/bbox.yaml

```diff
@@ -2,11 +2,17 @@ name: bbox
 in: query
 description: >-
   Only features that have a geometry that intersects the bounding box are selected. The bounding box is provided as four numbers:
-   
-  * Lower corner, coordinate axis 1 (minimum longitude)
-  * Lower corner, coordinate axis 2 (minimum latitude)
-  * Upper corner, coordinate axis 1 (maximum longitude)
-  * Upper corner, coordinate axis 2 (maximum latitude)
+
+  * Left box edge, coordinate axis 1
+  * Bottom box edge, coordinate axis 2
+  * Right box edge, coordinate axis 1
+  * Top box edge, coordinate axis 2
```

Comment:
I _presume_ the coordinate axis order is CRS-dependent? So Axis 1 could either be an Easting _or_ Northing?

In that case, maybe "Lower left corner, coordinate axis 1" & "Lower left corner, coordinate axis 2" is more accurate? (cf Upper right)

Or "SouthWest" & "NorthEast"?

Reply:
Yes, I was struggling how to best write it considering that 

* the axis order of the CRS applies and
* WGS84 longitude/latitude is the only CRS that we discuss in the core.

It would be nice, if we can find a wording that is unambiguous, but not (too) confusing for developers that are not familiar with coordinate reference systems. I agree, "lower left corner"/"upper right corner" is better.